src/app/cube/details/version-switch/version-switch.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="inner">
+  <div *ngIf="hasRevisions; then revisions; else noRevisions"></div>
```

Comment:
We should consider moving logic and copy specific to this issue outside of the component and keeping this component generic. That way, next time we need a switch somewhere, we have a component that's JUST a switch and we don't have to do anything extra.

Reply:
Resolved with Make switch more generic. It's only a switch now. The only thing that would have to get changed when using it somewhere else is the color of the toggle.